src/js/indexes/components/File.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styled from "styled-components";
+import { getFontWeight } from "../../app/theme";
+import { BoxGroupSection } from "../../base";
+import { byteSize } from "../../utils/utils";
+
+const StyledFile = styled(BoxGroupSection)`
+    display: flex;
+`;
+
+const StyledSize = styled.div`
+    margin-left: auto;
+    font-weight: ${getFontWeight("bold")};
+`;
+
+export const File = ({ fileName, download_url, size }) => {
```

Comment:
I think there is opportunity to share this with the file list in the subtraction detail view. Can you look at doing this?